{"uuid":"171dd0ca-af2f-432b-82a0-d6868c950d9d","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["65fd902e-b051-45a1-abbe-cc527a6b2740"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"22b56137-9a63-4a01-90e5-e5458d767311-attachment","type":"image/png"},{"name":"Myscreenshot","source":"b902eee9-c43e-437b-ac19-34a422f5a29d-attachment"}],"parameters":[],"start":1696883658189,"stop":1696883658235},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883658235,"stop":1696883658235},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883658236,"stop":1696883658236},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696883658236,"stop":1696883658236}],"start":1696883658186,"stop":1696883658237}